### PR TITLE
Fixed json_result unavailable error

### PR DESCRIPTION
**Pull request by femdom**
json_result was unavailable when getting error response if passing stream=True

### CODE REVIEW

File: cloudconvert/api.py [hunk 1]
```diff
@@ -146,35 +146,27 @@ def rawCall(self, method, path, content=None, is_authenticated=False, stream=Fal
 
         code = result.status_code
 
-
-        # attempt to decode and return the response
-
-        if not stream:
```
Comment (femdom) [inline cloudconvert/api.py]: 1. Imagine that stream=True

File: cloudconvert/api.py [hunk 2]
```diff
@@ -146,35 +146,27 @@ def rawCall(self, method, path, content=None, is_authenticated=False, stream=Fal
...
-            try:
-                json_result = result.json()
-            except ValueError as error:
-                raise InvalidResponse("Failed to decode API response", error)
-
-
         # error check
         if code >= 100 and code < 300:
```
Comment (femdom) [inline cloudconvert/api.py]: 2. Code == 400 for example

File: cloudconvert/api.py [hunk 3]
```diff
@@ -146,35 +146,27 @@ def rawCall(self, method, path, content=None, is_authenticated=False, stream=Fal
...
-            else:
-                return json_result
+
+            try:
+                return result.json()
+            except ValueError as error:
+                raise InvalidResponse("Failed to decode API response", error)
         else:
+            json_result = result.json()
             msg = json_result.get('message') if json_result.get('message') else json_result.get('error')
```
Comment (femdom) [inline cloudconvert/api.py]: 3. json_result is unavailable here

**Comment by josiasmontag** [comment]
Thanks and sorry for the delay!

**Pull request merged**